ibex_core.core

```diff
@@ -77,7 +82,7 @@ parameters:
 targets:
   default:
     filesets:
-      - tool_verilator ? (files_lint_verilator)
+      - tool_verilator ? (files_lint_verilator_all)
```

Comment:
Only include the `verilator_waiver.vlt` here (as it was before).

The other waiver needs to be included in the other systems. If you want to keep it in the current directory (`lint/verilator_waiver_other.vlt`), you need to create a separate `lint/xxx.core` file for it with a core named e.g. `lowrisc:ibex:waivers_other`, and depend on it in the simple system and compliance test cores). (You cannot have a file path starting with `..` in a core file.)

Reply:
Ah, right, I understand. I've just pushed up another version of the patch set. The first commit adds waiver files in `examples/simple_system/lint` and `dv/riscv_compliance/lint` and wires everything up. I hope I've got the fusesoc incantations right?